src/time_series_export/backend/influx_db9.cpp

```diff
@@ -0,0 +1,116 @@
+#include <string>
+#include <sstream>
+
+#include "curl/curl.h"
+#include "curl_easy.h"  // NOLINT(build/include)
+
+#include "glog/logging.h"
+
+#include "time_series_export/backend/influx_db9.hpp"
+
+#include "serenity/metrics_helper.hpp"
+
+namespace mesos {
+namespace serenity {
+
+using curl::curl_easy;
+
+void InfluxDb9Backend::PutMetric(const TimeSeriesRecord& _tsRecord) {
+  curl_easy easy = prepareRequest(_tsRecord);
+
+  try {
+    easy.perform();
+  }
+  catch (curl_easy_exception error) {
+    LOG(ERROR) << "InfluxDB9Backend: Error while inserting metrics, "
+               << error.what();
+  }
+}
+
+const curl_easy InfluxDb9Backend::prepareRequest(
+    const TimeSeriesRecord& _tsRecord) const {
+  curl_easy easy;
+
+  std::string url = getDbUrl();
+  std::string userpwd = getUserAndPassword();
+  std::string content = serializeRecord(_tsRecord);
+
+  easy.add(curl_pair<CURLoption, std::string>(CURLOPT_URL, url));
+
+  easy.add(curl_pair<CURLoption, uint64_t>(
+      CURLOPT_HTTPAUTH, CURLAUTH_BASIC));
+  easy.add(curl_pair<CURLoption, std::string>(CURLOPT_USERPWD, userpwd));
+
+  easy.add(curl_pair<CURLoption, int64_t>(CURLOPT_POST, 1));
+  easy.add(curl_pair<CURLoption, int64_t>(CURLOPT_POSTFIELDSIZE,
+                                          content.size()));
+  easy.add(curl_pair<CURLoption, std::string>(CURLOPT_POSTFIELDS, content));
+
+  return easy;
+}
+
+
+/**
+ * Returns URL to database.
```

Comment:
s/Returns/Returns/ (see method below - you wrote `serialize` not `serializes`, so change here or `serializes`)